src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java

```diff
@@ -315,6 +318,28 @@ ExitCode parseOptions(List<String> args, ExtendedEventHandler eventHandler) {
     return ExitCode.SUCCESS;
   }
 
+  /**
+   * If --enable_platform_specific_config is true and the corresponding config definition exists,
+   * we should enable the platform specific config.
+   */
+  private boolean shouldEnablePlatformSpecificConfig(
+      OptionValueDescription enablePlatformSpecificConfigDescription,
+      ListMultimap<String, RcChunkOfArgs> commandToRcArgs) {
+    if (enablePlatformSpecificConfigDescription == null ||
+        !(boolean) enablePlatformSpecificConfigDescription.getValue()) {
```

Comment:
Is this cast safe? Shouldn't it be `(Boolean)` and check for null?

Reply:
Yes, as long as there is a default value, it won't be null and will be converted to the correct type. 
See https://github.com/bazelbuild/bazel/blob/1f684e1b87cd8881a0a4b33e86ba66743e32d674/src/main/java/com/google/devtools/common/options/OptionDefinition.java#L255

Reply:
Thanks!